packages/babel-runtime/package.json

```diff
@@ -18,166 +18,406 @@
   },
   "exports": {
     "./helpers/": "./helpers/",
```

Comment:
the order of the keys is important here - are you sure this won't just always be matched before following conditions for specific helpers?

Reply:
To be safe we can probably just use `./helpers/esm/` here, since it's the only thing not covered by all the other patterns.

Reply:
Makes sense, although I would probably put it below either way - any forwarding condition acts as a "catch-all" (for a given path), and as such, I think it should be below - it's like a default case for a switch statement 😅 

I'm curious though if this has indeed interfered with the algorithm 🤔

Reply:
First of all, thanks for making me check the docs because it looks like this kind of `exports` will be removed :grimacing: 

https://nodejs.org/api/packages.html#packages_subpath_folder_mappings

Reply:
> First of all, thanks for making me check the docs because it looks like this kind of exports will be removed 😬

I'll try to refrain from snarky comments about how this got shipped to node...

> Also, https://nodejs.org/api/packages.html#packages_conditional_exports says that the first pattern that matches is used.

Right, that was what prompted my comment.

> I can surprise you 😉

Not for the first time 😅 During the day I had a feeling that your Twitter posts were prompted by this conversation 😉